editoast/src/models/fixtures.rs

```diff
@@ -214,6 +221,62 @@ pub async fn create_rolling_stock_livery(
         .expect("Failed to create rolling stock livery")
 }
 
+pub fn create_towed_rolling_stock() -> TowedRollingStock {
```

Comment:
Maybe there is a possible factorization with a similar function [here](https://github.com/OpenRailAssociation/osrd/blob/a54923883baad47b9803c4dc97794176481e4139/editoast/src/views/rolling_stock/towed.rs#L363-L389).